ext/net/lib.deno_net.d.ts

```diff
@@ -76,6 +76,9 @@ declare namespace Deno {
      * You should show the message like `server running on localhost:8080` instead of
      * `server running on 0.0.0.0:8080` if your program supports Windows. */
     hostname?: string;
+    /** SO_REUSEADDR option for socket.
+     * If not specified, defaults to true in Linux, false in Windows. */
```

Comment:
**SO_REUSEADDR** defaults to true because, as stated in #13808, the previous implementation of TCP socket used the [std::net](https://github.com/rust-lang/rust/blob/6343edfa11c80ffed02de7e059178d4c1637ecdb/library/std/src/sys_common/net.rs) implementation, which sets **SO_REUSEADDR** to true when binding a **TcpListener** outside of Windows. In this way it is possible to keep the behavior of existing code that does not explicit use the new **reuseAddress** argument. Setting this to **false** as default value could break existing code due to the fact that in Unix system is not possible to bind socket to a combination of address+port occupied by closed sockets in the **TIME_WAIT** state, as visible in failed tests in #13794. In addition, this seem the same behavior of node: [unix tcp](https://github.com/nodejs/node/blob/e46c680bf2b211bbd52cf959ca17ee98c7f657f5/deps/uv/src/unix/tcp.c), [win tcp](https://github.com/nodejs/node/blob/e46c680bf2b211bbd52cf959ca17ee98c7f657f5/deps/uv/src/win/tcp.c).